### DISCUSSION TITLE
Application Insights for your blazor wasm project

### DISCUSSION THREAD

**Discussion by SoucianceEqdamRashti** [jasontaylordev/CleanArchitecture#478] [Q&A] [answered]
Hello,

I saw your great presentation "Modern Web Dev with Blazor and .NET 6 with Jason Taylor" on youtube. However, one thing that I am having trouble with when it comes to release blazor apps to production is how to use monitoring and loggng and especially application insights. There is no out of hte box sdk for it so how do you monitor or centralize your logs for your blazor wasm apps? Any hints on this would be extremely helpful.

Thanks
Souciance

**Comment by jasontaylordev** [comment, marked as answer]
Check out this project - https://github.com/IvanJosipovic/BlazorApplicationInsights

**Reply by wicksipedia** [reply]
we just used this on a client project - client was very happy

**Reply by SoucianceEqdamRashti** [reply]
Yeah I have started to use this project as well. I do hope something comes that is out of the box but will have to be good for now.

Also a tip for the next video maybe some things regarding best practises and antipatterns for Blazor?
